straxen/contexts.py

```diff
@@ -62,33 +62,26 @@
 # XENONnT
 ##
 
-
 def xenonnt_online(output_folder='./strax_data',
                    we_are_the_daq=False,
                    **kwargs):
     """XENONnT online processing and analysis"""
-    if we_are_the_daq:
-        run_db_username = straxen.get_secret('mongo_rdb_username')
-        run_db_password = straxen.get_secret('mongo_rdb_password')
-        mongo_url = f"mongodb://{run_db_username}:{run_db_password}@xenon1t-daq:27017,old-gw:27017/admin"
-    else:
-        mongo_url = None   # Default URL should work
-
     context_options = {
         **straxen.contexts.common_opts,
         **kwargs}
 
     st = strax.Context(
         storage=[
             straxen.RunDB(
-                mongo_url=mongo_url,
-                runid_field='number',
                 readonly=not we_are_the_daq,
+                runid_field='number',
                 new_data_path=output_folder),
             strax.DataDirectory(
                 '/dali/lgrandi/xenonnt/raw',
                 readonly=True,
-                take_only='raw_records')],
+                take_only='raw_records'),
+            strax.DataDirectory(output_folder),
```

Comment:
Good point, can fix. But I didn't expect that particular error: you got this from bootstrax or microstrax?

Reply:
It was bootstrax. I also wouldn't be expecting microstrax to process raw_records

Reply:
Yes, microstrax shouldn't do any processing, so that's good. I tried to fix now, but feel free to fix it more / commit to master if this doesn't cut it.

Reply:
Thanks Jelle, looks good.